src/doc/unstable-book/src/compiler-flags/sanitizer.md

```diff
@@ -25,6 +25,8 @@ This feature allows for use of one of following sanitizers:
   * [LeakSanitizer](#leaksanitizer) a run-time memory leak detector.
   * [MemorySanitizer](#memorysanitizer) a detector of uninitialized reads.
   * [ThreadSanitizer](#threadsanitizer) a fast data race detector.
+  * [RealtimeSanitizer](#realtimesanitizer) a detector of blocking functions
```

Comment:
I would maybe go with "non-deterministic execution time calls in real-time contexts". 

(as a side note in our docs and presentations we were trying to go with real-time over realtime. This is a matter of preference but it would be good to stick with what we decided back then)